docs/primehub-app-tutorial-label-studio.md

```diff
@@ -69,4 +69,59 @@ It will open a new window and show the Label Studio UI. You can find your login
 7. After you submit the labeled result, the labeled json file will be under the `/project/<group_name>/dog-demo-labeled`.
     ![](assets/app_tutorial_labelstudio_labeled_json.png)
 
-That's the basic use of how to label the dataset by using Label Studio and PrimeHub. Enjoy it!
\ No newline at end of file
+That's the basic use of how to label the dataset by using Label Studio and PrimeHub. Enjoy it!
+
+## How to Use Labeled Data to Train a Model
+
+In the last section, we show you how to label the dataset. Now, we want to demonstrate how you can use the labeled data to train a model.
+
+For simplicity, the model will be a classification model and you also only need to label the class of the image. The model classifies whether the screw is good or bad.
+
+Here are examples of good and bad screws. The first image is the good screw. The second image is the bad screw and you can see the there is a manipulated front.
+
+![](assets/app_tutorial_labelstudio_screw_good.png)
+![](assets/app_tutorial_labelstudio_screw_bad.png)
+
+### What we need?
+
+- Create a [dataset](guide_manual/admin-dataset) in PrimeHub called `screw`, and set the read/write permission to your group. Please download the [app_tutorial_labelstudio_screw_dataset.zip](assets/app_tutorial_labelstudio_screw_dataset.zip), unzip it and upload images to the `~/datasets/screw` folder by the [notebook](quickstart/launch-project)
+- Create a directory `/project/<group_name>/screw-labeled` in group volume to save the labeled results
+- The image `infuseai/docker-stacks:pytorch-notebook-b3c52f35`
```

Comment:
this image will sometimes show 'Build Recommended' prompt, maybe change to use another image?
![i](https://user-images.githubusercontent.com/16533561/119600755-4a4e3a80-be1a-11eb-831d-b717e275318d.png)